src/app/_components/ListboxOptions.tsx

```diff
@@ -11,20 +11,30 @@ type HeadlessUIAnchorProps = NonNullable<
 >
 type Position = Extract<HeadlessUIAnchorProps, string>
 
-type ListboxOptionsProps = {
+export type ListboxOptionsProps = {
   as?: ElementType
   position?: Position
-  matchButtonWidth?: boolean
+  width?: keyof typeof widthClasses
   children: React.ReactNode
 }
 
 const GAP_BETWEEN_BUTTON_AND_OPTIONS_IN_PX = 8
 
+const widthClasses = {
+  auto: 'w-auto',
+  button: 'w-[var(--button-width)]',
+  xs: 'w-[16rem]',
+  sm: 'w-[20rem]',
+  md: 'w-[24rem]',
+  lg: 'w-[28rem]',
+  xl: 'w-[32rem]',
+} as const
```

Comment:
Let's see if we still can revert this when we use the abbreviations -`Dapp`, `NFTs`, `DAO`. We can open a new PR for the renaming :-)